radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/InMemoryAtomStore.java

```diff
@@ -22,6 +23,11 @@
 	 */
 	private final ConcurrentHashMap<RadixAddress, ReplaySubject<AtomObservation>> cache = new ConcurrentHashMap<>();
 
+	/**
+	 * The In Memory Atom Data Store
```

Comment:
This doesn't seem like a very apt comment here.

Reply:
lol oops